pkg/build/admission/defaults/v1/types.go

```diff
@@ -0,0 +1,21 @@
+package v1
+
+import (
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+)
+
+// BuildDefaultsConfig controls the default information for Builds
+type BuildDefaultsConfig struct {
```

Comment:
We need to use the config api scheme (pkg/cmd/server/api) and not the kube api scheme so that when the master config is deserialized we can read these plugin configs.